templates/modcontrol/contentbyuser.html

```diff
@@ -35,11 +35,28 @@
             </div>
           </td>
           <td class="thumb-cell">
-            $if 'sub_media' in i:
-              $ thumb = THUMB(i)
-              <img src="${thumb['display_url']}" alt="" class="thumb" />
+            $# custom thumb, if applicable
+            $if i['contype'] == 10 and 'thumbnail-custom' in i['sub_media']:
+              <img src="${i['sub_media']['thumbnail-custom'][0]['display_url']}" alt="" class="thumb" />
+            $elif i['contype'] == 20:
+              $# for characters, all thumbnails show as 'generated'.. even if they are custom..
+              <img src="${i['sub_media']['thumbnail-generated'][0]['display_url']}" alt="" class="thumb" />
+            $elif i['contype'] == 30:
+              &nbsp;
```

Comment:
Is this just for the `else`? `$elif i['contype'] != 30:` doesn’t read too badly.